### PR TITLE
[T2A3] [W09_A2] Nguyen Quoc Bao

### PR DESCRIPTION
**Pull request by bqnguyen94**

### CODE REVIEW

**Comment by bqnguyen94** [comment]
Ready for review.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -269,7 +282,7 @@ private static void processProgramArgs(String[] args) {
         if(args.length == 0) {
             setupDefaultFileForStorage();
         }
-    }
+    }*/
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/AddressBook.java:285]: Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.

**Comment by jeffryhartanto** [comment]
@bqnguyen94  Some comments added. Please close the PR after reading comments.

**Comment by jeffryhartanto** [comment]
@bqnguyen94 Some comments added. Please close the PR after reading comments. Don't forget to change your prof pic.

**Comment by jeffryhartanto** [comment]
@bqnguyen94 Some comments added. Please close the PR after reading comments. Good job for doing the assignment, make sure next time each commit address one task only.

**Pull request closed without merging**